smaug/core/node.proto

```diff
@@ -60,9 +60,8 @@ message NodeProto {
   // Unique node name in the form of "scopes:name". Scopes are optional.
   string name = 1;
   OpType op = 2;
-  // Parents and children of this node.
+  // Parents of this node.
```

Comment:
It's true that we only *need* a singly-linked list, but having a backwards pointer makes a lot of algorithms a lot easier to write. Are you sure this is what you want?

Reply:
Yes, I want to remove this as I find maintaining the children of a node is actually quite some work when we start supporting graph post-processing like inserting switch nodes for inputs.